src/features/query-bar/components/filter/Condition.js

```diff
@@ -176,12 +178,52 @@ const Condition = ({
         );
     };
 
+    const renderAttributeField = () => {
+        const { columnKey } = condition;
+        const columnAttributes = columns || [];
+        const attributeOptions = getFormattedOptions(columnAttributes);
+
+        return (
+            <div className="filter-item-attribute-dropdown-container">
+                <div className="filter-dropdown-single-select-field-container">
+                    <SingleSelectField
+                        fieldType={ATTRIBUTE}
+                        isDisabled={false}
+                        onChange={updateSelectedField}
+                        options={attributeOptions}
+                        placeholder={formatMessage(messages.selectAttributePlaceholderText)}
+                        selectedValue={columnKey}
+                    />
+                </div>
+            </div>
+        );
+    };
+
+    const renderOperatorField = () => {
+        const { operatorKey } = condition;
+        const operatorsForAttribute = getOperatorsForAttribute();
+        const operatorOptions = getFormattedOptions(operatorsForAttribute);
+
+        return (
+            <div className="filter-item-operator-dropdown-container">
+                <div className="filter-dropdown-single-select-field-container">
```

Comment:
You can potentially save some bytes by making the class a constant. I see it's being used twice and its long.
Also is double-nested div needed?

Reply:
I can address in a separate PR
No it looks like the double nested div is not needed